packages/workbox-background-sync/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "workbox-background-sync",
+  "version": "3.0.0",
+  "license": "Apache-2.0",
+  "author": "Google's Web DevRel Team",
+  "description": "Queues failed requests and uses the Background Sync API to replay them when the network is available",
+  "repository": "googlechrome/workbox",
+  "bugs": "https://github.com/googlechrome/workbox/issues",
+  "homepage": "https://github.com/GoogleChrome/workbox",
+  "keywords": [
+    "workbox",
+    "workboxjs",
+    "service worker",
+    "sw",
+    "background",
+    "sync"
+  ],
+  "scripts": {
+    "prepublish": "gulp build-packages --package workbox-background-sync"
```

Comment:
This should be changed to "build" and add "prepare" and "version" npm run scripts to call `npm run build` see modules in v3 branch.

Reply:
Done.